### PR TITLE
fix(core): warn when vuex store is not activated

### PR DESCRIPTION
**Pull request by JoaoPedroAS51**

### CODE REVIEW

File: src/core/storage.ts [hunk 1]
```diff
@@ -131,6 +131,11 @@ export class Storage {
       this.state = this.ctx.store.state[this.options.vuex.namespace]
     } else {
       Vue.set(this, 'state', {})
+
+      // eslint-disable-next-line no-console
+      console.warn(
+        '[AUTH] The Vuex Store is not activated. To activate it, see https://nuxtjs.org/docs/2.x/directory-structure/store'
```
Comment (bmulholland) [inline src/core/storage.ts:137]: This should state why it's a problem. Think about the scenario where you're a developer implementing this, and you see the warning - why do you care? Or when you're debugging the problem this causes - why pay attention to this warning? Try to answer that in the warning text.

**Review by bmulholland — CHANGES_REQUESTED** [review]
Can improve messaging

File: src/core/storage.ts (hunk 1 above)
Comment (JoaoPedroAS51) [reply src/core/storage.ts:137]: Ok, got it! :)

Comment (JoaoPedroAS51) [reply src/core/storage.ts]: Done

**Pull request merged**